src/scripts/modules/media/body/body.less

```diff
@@ -65,7 +65,7 @@
 
     .media-body-about-cover {
       width: ~"calc(125px + 2rem)";
-      height: intrinsic;
+      height: ~"calc(125px + 2rem)";
```

Comment:
Could the height just be omitted, rather than forcing the image to be a square?

Reply:
no, it ends up all kinds of janky heights because the browser sometimes sets its height based on the un-flexy width of the image